init_suite.py

```diff
@@ -54,22 +57,27 @@ def process_override(dir_name: str) -> List:
     # At this point, the entry/item is a directory. Inside this folder, we
     # support only three files
     override_data = dict()
-    test_data = dict()
+    #test_data = dict()
+    test_data = dict({'tests':list()})
 
     log.debug(f"Processing overrides of {dir_name}")
 
     files = glob(os.path.join(dir_name, "*"))
+    files.sort()
     for file in files:
         if file.endswith("overrides.yaml"):
             override_data = read_yaml(file)
             continue
 
-        test_data = read_yaml(file)
+        #test_data = read_yaml(file)
```

Comment:
Please remove the line if it is not be used.

Reply:
okay